README.md

```diff
@@ -1 +1,148 @@
-# metrics-webapp
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>UK Carbon Intensity Indicator</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Author](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 UK Carbon Intensity Indicator <a name="about-project"></a>
+
+**UK Carbon Intensity Indicator** is a web app built in React/Redux that displays Carbon Intensity forecasts for the United Kingdom.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="#">React/Redux</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+- Displays carbon intensity forecasts for each region
+- Filters to view regions with specific intensity indices
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live demo](https://carbon-intensity.onrender.com/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+Git
+Code Editor
+Any HTML5-compliant browser
+NPM package manager
+
+### Setup
+
+Clone this repository to your local folder: https://github.com/balatstar/metrics-webapp.git
+
+
+### Install
+
+Install this project with:
+
+git clone https://github.com/balatstar/metrics-webapp.git
+Open directory with your preferred code editor
+Run: npm install
+
+### Usage
+
+To run the project, execute the following command:
+
+npm start
+
+### Run tests
+
+To run tests, run the following command:
+
+npm test
+
+### Deployment
+
+You can deploy this project using:
+
+npm install
+
```

Comment:
- [ ] Please be sure to enter the appropriate information to this section. The `npm install` project command is used to install the project's dependencies.😊